src/pages/kitae/Login/LoginKitae.scss

```
@@ -0,0 +1,88 @@
+body {
+  background-color: #fafafa;
+  margin: 0px;
+  color: #262626;
+  box-sizing: borderBox;
+  .loginWrapper {
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 350px;
+    height: 380px;
+    top: 45%;
+    left: 50%;
+    margin: -175px 0px 0px -190px;
+    background-color: white;
+    border: 1px solid #dbdbdb;
+
+    .loginLogo {
+      margin: 30px 0 30px 0;
+      color: #262626;
+      font-family: 'Lobster', cursive;
+      font-size: 40px;
+      font-weight: 100px;
+    }
+
+    .loginForm {
+      display: flex;
+      flex-direction: column;
+      width: 280px;
+      margin-top: 10px;
+
+      .loginText {
+        width: 100%;
+        height: 40px;
+        margin: 3px 0 3px 0;
+        padding: 0 10px 0 10px;
+        background-color: #fafafa;
+        border: 1px solid #dbdbdb;
+        border-radius: 3px;
+      }
+
+      .loginText:focus {
+        outline: 1px solid #a8a8a8;
+      }
+
+      .loginText::placeholder {
+        color: #8e8e8e;
+      }
```

```suggestion
      .loginText {
        width: 100%;
        height: 40px;
        margin: 3px 0 3px 0;
        padding: 0 10px 0 10px;
        background-color: #fafafa;
        border: 1px solid #dbdbdb;
        border-radius: 3px;
        
      &:focus {
        outline: 1px solid #a8a8a8;
      }

      &::placeholder {
        color: #8e8e8e;
      }
```

nesting과 & 연산자를 활용해서 구현해주세요!